javarosa/src/main/java/org/javarosa/core/util/SizeBoundVector.java

```diff
@@ -34,26 +30,14 @@ public int getAdditional() {
     }
 
     public void addBadImageReference() {
-        badImageReferenceCount++;
+        // TODO: remove this
```

Comment:
It is sorta hard to fully remove it because it is used by the core versions of the suite/xform installer's `verifyInstallation`. That code isn't run currently on the CLI, but we might want to enable it. Figuring out the right path forward seems like way more work than I want to do for this right now. I can revert the changes in this file if you'd like.